internal/strict/strict.go

```diff
@@ -103,6 +106,10 @@ var StrictControls = Controls{
 		Error:   errors.New("The `validate-all` command is no longer supported. Use `terragrunt run-all validate` instead."),
 		Warning: "The `validate-all` command is deprecated and will be removed in a future version. Use `terragrunt run-all validate` instead.",
 	},
+	SkipDependenciesInputs: {
+		Error:   errors.New("The `skip-dependencies-inputs` option is deprecated. Reading inputs from dependencies has been deprecated and will be removed in a future version of Terragrunt. To continue using inputs from dependencies, forward them as outputs."),
+		Warning: "The `skip-dependencies-inputs` option is deprecated and will be removed in a future version of Terragrunt. Reading inputs from dependencies has been deprecated. To continue using inputs from dependencies, forward them as outputs.",
```

Comment:
```suggestion
		Error:   errors.Errorf("The `%s` option is deprecated. Reading inputs from dependencies has been deprecated and will be removed in a future version of Terragrunt. To continue using inputs from dependencies, forward them as outputs.", SkipDependenciesInputs),
		Warning: fmt.Errorf("The `%s` option is deprecated and will be removed in a future version of Terragrunt. Reading inputs from dependencies has been deprecated. To continue using inputs from dependencies, forward them as outputs.",  SkipDependenciesInputs),
```

Will help to avoid problems in case of renaming.

Reply:
Updated